applications/FluidDynamicsBiomedicalApplication/custom_utilities/wss_statistics_utilities.cpp

```diff
@@ -0,0 +1,151 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Eduardo Soudah,
+//                   Ruben Zorilla
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/variables.h"
+#include "utilities/body_normal_calculation_utils.h"
+#include "utilities/parallel_utilities.h"
+#include "utilities/variable_redistribution_utility.h"
+#include "utilities/variable_utils.h"
+
+// Application includes
+#include "wss_statistics_utilities.h"
+#include "fluid_dynamics_biomedical_application_variables.h"
+
+namespace Kratos {
+
+void WssStatisticsUtilities::InitializeWSSVariables(ModelPart& rModelPart)
+{
+    // Initialize WSS variables
+    const array_1d<double,3> aux_zero = ZeroVector(3);
+    block_for_each(rModelPart.Nodes(), [&aux_zero](NodeType& rNode){
+        rNode.SetValue(WSS, 0.0);
+        rNode.SetValue(RRT, 0.0);
+        rNode.SetValue(OSI, 0.0);
+        rNode.SetValue(ECAP, 0.0);
+        rNode.SetValue(TWSS, 0.0);
+        rNode.SetValue(TAWSS, 0.0);
+        rNode.SetValue(TEMPORAL_OSI, aux_zero);
+        rNode.SetValue(WSS_NORMAL_STRESS, aux_zero);
+        rNode.SetValue(WSS_TANGENTIAL_STRESS, aux_zero);
+    });
+}

```

Comment:
Why a separate method? Can't we integrate these `SetValue` methods in corresponding methods where they are set? That way there is less room for errors.

Reply:
See my comment above.